### PR TITLE
Update ExpressionBuilder.php

### PR DESCRIPTION
**Pull request by rabihsyw**
parse array values as strings

### CODE REVIEW

**Comment by Ocramius** [comment]
This patch needs additional tests and a better description. Also, check existing tests (they seem to fail on travis-ci)

File: tests/Doctrine/Tests/DBAL/Query/Expression/ExpressionBuilderTest.php [hunk 1]
```diff
@@ -199,7 +199,7 @@ public function testIsNotNull()
 
     public function testIn()
     {
-        $this->assertEquals('u.groups IN (1, 3, 4, 7)', $this->expr->in('u.groups', array(1,3,4,7)));
+        $this->assertEquals("u.groups IN ('1', '3', '4', '7')", $this->expr->in('u.groups', array(1, 3, 4, 7)));
```
Comment (Ocramius) [inline tests/Doctrine/Tests/DBAL/Query/Expression/ExpressionBuilderTest.php]: This test shouldn't have changed

File: tests/Doctrine/Tests/DBAL/Query/Expression/ExpressionBuilderTest.php [hunk 2]
```diff
@@ -209,7 +209,7 @@ public function testInWithPlaceholder()
 
     public function testNotIn()
     {
-        $this->assertEquals('u.groups NOT IN (1, 3, 4, 7)', $this->expr->notIn('u.groups', array(1,3,4,7)));
+        $this->assertEquals("u.groups NOT IN ('1', '3', '4', '7')", $this->expr->notIn('u.groups', array(1, 3, 4, 7)));
```
Comment (Ocramius) [inline tests/Doctrine/Tests/DBAL/Query/Expression/ExpressionBuilderTest.php]: This test shouldn't have changed

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by morozov** [comment]
The API is improperly documented. Similar to the majority of the similar `AbstractPlatform` methods, `in()` expects an array of SQL expressions, not values.

**Pull request closed without merging**